File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TripResponseTrips.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.TripResponseEndCoordinates
import org.openapitools.model.TripResponseStartCoordinates
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param endOdometer Odometer reading at the end of the trip.
 * @param distanceMeters Length of the trip in meters.
 * @param endMs End of the trip in UNIX milliseconds.
 * @param startMs Beginning of the trip in UNIX milliseconds.
 * @param fuelConsumedMl Amount in milliliters of fuel consumed on this trip.
 * @param startAddress Text representation of nearest identifiable location to the start (latitude, longitude) coordinates.
 * @param startCoordinates 
 * @param endCoordinates 
 * @param startOdometer Odometer reading at the beginning of the trip.
 * @param driverId ID of the driver.
 * @param startLocation Geocoded street address of start (latitude, longitude) coordinates.
 * @param tollMeters Length in meters trip spent on toll roads.
 * @param endAddress Text representation of nearest identifiable location to the end (latitude, longitude) coordinates.
 * @param endLocation Geocoded street address of start (latitude, longitude) coordinates.
 */
data class TripResponseTrips (

        @JsonProperty("endOdometer") val endOdometer: Int? = null,

        @JsonProperty("distanceMeters") val distanceMeters: Int? = null,

        @JsonProperty("endMs") val endMs: Int? = null,

        @JsonProperty("startMs") val startMs: Int? = null,

        @JsonProperty("fuelConsumedMl") val fuelConsumedMl: Int? = null,

        @JsonProperty("startAddress") val startAddress: String? = null,

        @JsonProperty("startCoordinates") val startCoordinates: TripResponseStartCoordinates? = null,

        @JsonProperty("endCoordinates") val endCoordinates: TripResponseEndCoordinates? = null,

        @JsonProperty("startOdometer") val startOdometer: Int? = null,

        @JsonProperty("driverId") val driverId: Int? = null,

        @JsonProperty("startLocation") val startLocation: String? = null,

        @JsonProperty("tollMeters") val tollMeters: Int? = null,

        @JsonProperty("endAddress") val endAddress: String? = null,

        @JsonProperty("endLocation") val endLocation: String? = null
) {

}

File: openapi-generator/php-ze-ph/src/App/DTO/TripResponseTrips.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class TripResponseTrips
{
    /**
     * Odometer reading at the end of the trip.
     * @DTA\Data(field="endOdometer", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $end_odometer;
    /**
     * Length of the trip in meters.
     * @DTA\Data(field="distanceMeters", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $distance_meters;
    /**
     * End of the trip in UNIX milliseconds.
     * @DTA\Data(field="endMs", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $end_ms;
    /**
     * Beginning of the trip in UNIX milliseconds.
     * @DTA\Data(field="startMs", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $start_ms;
    /**
     * Amount in milliliters of fuel consumed on this trip.
     * @DTA\Data(field="fuelConsumedMl", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $fuel_consumed_ml;
    /**
     * Text representation of nearest identifiable location to the start (latitude, longitude) coordinates.
     * @DTA\Data(field="startAddress", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $start_address;
    /**
     * @DTA\Data(field="startCoordinates", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\TripResponseStartCoordinates::class})
     * @DTA\Validator(name="Dictionary", options={"type":\App\DTO\TripResponseStartCoordinates::class})
     * @var \App\DTO\TripResponseStartCoordinates
     */
    public $start_coordinates;
    /**
     * @DTA\Data(field="endCoordinates", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\TripResponseEndCoordinates::class})
     * @DTA\Validator(name="Dictionary", options={"type":\App\DTO\TripResponseEndCoordinates::class})
     * @var \App\DTO\TripResponseEndCoordinates
     */
    public $end_coordinates;
    /**
     * Odometer reading at the beginning of the trip.
     * @DTA\Data(field="startOdometer", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $start_odometer;
    /**
     * ID of the driver.
     * @DTA\Data(field="driverId", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $driver_id;
    /**
     * Geocoded street address of start (latitude, longitude) coordinates.
     * @DTA\Data(field="startLocation", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $start_location;
    /**
     * Length in meters trip spent on toll roads.
     * @DTA\Data(field="tollMeters", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $toll_meters;
    /**
     * Text representation of nearest identifiable location to the end (latitude, longitude) coordinates.
     * @DTA\Data(field="endAddress", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $end_address;
    /**
     * Geocoded street address of start (latitude, longitude) coordinates.
     * @DTA\Data(field="endLocation", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $end_location;
}
